templates/base.html

```diff
@@ -43,6 +43,12 @@
 	<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/languages/scala.min.js"
 			integrity="sha512-F7HBSjTYDsAreqced8cggvF5AuB+j2TXktnLYpCrpWF2fhQJJCc5KimR3LYLs0AppTBecbPN9M9Hjf8yC2WstA=="
 			crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.8.0/languages/matlab.min.js"
```

Comment:
Now that we have already three extra languages added to the default bundle, it starts to make sense to build highlight.js locally and include the used languages there. Then, we only need to add one highlight.js file in the HTML templates.

This is a different task for the future. We already have an issue for it: #503 (that is, extend the drone.io pipeline with more libraries; Bootstrap is already there).